deepfence_frontend/apps/dashboard/src/features/postures/data-component/toggleControlApiAction.ts

```diff
@@ -40,7 +41,8 @@ export const toggleControlApiAction = async ({
     });
     if (!result.ok) {
       if (result.error.response.status === 400 || result.error.response.status === 409) {
-        toast.error(result.error.message);
+        const { message } = await getResponseErrors(result.error);
+        toast.error(message);
```

Comment:
Should there be a return statement here?

Reply:
Yes there is missing return statement, also added in one more place where it is missing.